src/openzaak/components/catalogi/admin/admin_views.py

```diff
@@ -80,6 +80,13 @@ class CatalogusZaakTypeImportSelectView(
     permission_required = "catalogi.add_zaaktype"
     raise_exception = True
 
+    @staticmethod
+    def get_query_dict(querset):
+        query_dict = {}
+        for obj in querset:
+            query_dict[obj.omschrijving] = obj.pk
+        return query_dict
```

Comment:
It can be replaced with one line with dict comprehension and I don't think there is a need in a special function for it